packages/puppeteer/esmTest.mjs

```diff
@@ -0,0 +1,7 @@
+import defaultExport, { AxePuppeteer } from './dist/index.mjs';
+import assert from 'assert';
+
+const exportIsFunction = typeof(defaultExport) === 'function';
+const exportIsSame = defaultExport === AxePuppeteer;
+assert(exportIsFunction, 'export is not a function');
+assert(exportIsSame, 'default and named export is not the same');
```

Comment:
Looks like we're missing the `exportIsSame` assertion for the other integrations, we should probably add those too

Reply:
I only added them when they made sense: when there is both a named and default export.